ir/pass_manager.cpp

```diff
@@ -41,7 +41,8 @@ const IR::Node *PassManager::apply_visitor(const IR::Node *program, const char *
                 program = program->apply(**it);
                 LOG3(log_indent << "heap after " << v->name() << ": in use " <<
                      n4(gc_mem_inuse(&maxmem)) << "B, max " << n4(maxmem) << "B");
-                int errors = ErrorReporter::instance.getErrorCount();
+                auto& context = BaseCompileContext::get();
+                int errors = context.errorReporter().getErrorCount();
```

Comment:
perhaps there should be a global `::getErrorCount()` (analogous to the global `::getDiagnosticCount()` that already exists)?

Reply:
There actually is, but I didn't update this call site to use it. =) I'll do that.